src/Table.luna

```diff
@@ -844,17 +1266,88 @@ class Table:
+    # >     table = Table.fromColumns [col1 , col2, col3, col4]
+    # >     rolling = table.rollingIntervalMultiple "col1" 2.seconds [("col2", [Mean, Std])]
+    # >     None
+    #
+    # `keyColumnName`: Text name for column which will be an indexing column.
+    # `interval`: Time interval for rolling window, i.e. `2.days`.
+    # `aggregations`: List of aggregations. Single aggregation is a two argument tuple. First element is a name of a column for aggregation and second is a list of aggregate functions.
+    #
+    # `return`: `Table` value with key column and resulting columns for each aggregation. 
+
     def rollingIntervalMultiple keyColumnName interval aggregations:
         keyColumnWrapper = self.column keyColumnName . ptr
         aggregationWithWrappers = aggregations.each (colname, aggrs): (self.column colname . ptr, aggrs.each .toInt)
         self.fromWrapper $ self.ptr.rollingInterval keyColumnWrapper (interval.toMicroseconds * 1000) aggregationWithWrappers
 
-    # rollingInterval :: Text -> TimeInterval -> Text -> AggregateFunction -> Table
+    # Provides rolling window calculations. Rolling is provided over `keyColumnName`. The windows size is defined by `interval` and the statistic operation over the values in column `aggregateColumnName` is defined by `aggregateFunction`.
+    #
+    # > import Dataframes.Column
+    # > import Dataframes.Types
+    # > import Dataframes.Table
+    # > import Std.Time
+    # > 
+    # > def main:
+    # >     l1 = ["2013-01-01 09:00:00", "2013-01-01 09:00:02", "2013-01-01 09:00:03", "2013-01-01 09:00:05", "2013-01-01 09:00:06"].each (Time.parseFmt "%F %T")
+    # >     l2 = [11,12,13,14,15]
+    # >     l3 = [21,22,23,24,25]
+    # >     l4 = [31,32,33,34,35]
+    # >     col1 = Column.fromList "col1" TimestampType l1
+    # >     col2 = Column.fromList "col2" Int64Type l2
+    # >     col3 = Column.fromList "col3" Int64Type l3
+    # >     col4 = Column.fromList "col4" Int64Type l4
+    # >     table = Table.fromColumns [col1 , col2, col3, col4]
+    # >     rolling = table.rollingInterval "col1" 2.seconds "col2" Mean
+    # >     None
+    #
+    # `keyColumnName`: Text name for column which will be an indexing column.
+    # `interval`: Time interval for rolling window, i.e. `2.days`.
+    # `aggregateColumnName`: Name of a column for aggregation.
+    # `aggregateFunction`:
+    #
+    # `return`: `Table` value with key column and resulting columns for each aggregation. 
     def rollingInterval keyColumnName interval aggregateColumnName aggregateFunction:
         self.rollingIntervalMultiple keyColumnName interval [(aggregateColumnName,[aggregateFunction])]
 
-    # shift :: Text -> Int -> Table
+    # Shift index by desired number of periods for selected column.
+    # 
+    # Newly created column is added at the end of the table. The index value can be both positive and negative. 
+    # Missing values in newly created column will be filled with `0`. 
```

Comment:
zroes? not nulls?

Reply:
our wording is not consistent: in header we talk about shifting the selected column (as if it was mutated), here we talk about "newly created column".
We should be consistent — either we describe APIs as-if they were mutating the data (and they are just returning copies) or not. Now user might be confused — newly created? I don't want to create anything, just to shift.